[CTOR-1720]-Refresh applications-databases-db-2 doc page (#4520)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-databases-db2.md

Reviewer: ```suggestion
### Install plugin dependencies
```

Reviewer: ```suggestion
In order to use this connector, the `wget` command-line tool and the GNU Compiler Collection (`gcc`) are necessary.
```

> Eventually, this user must have the authority `SYSMON`.

Reviewer: ```suggestion
This user must have the permission to execute the `SYSPROC.GET_DBSIZE_INFO` procedure and the `SYSMON` authority.
```

| EXTRAOPTIONS           | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                                          | --verbose               |             |

</TabItem>
</Tabs>

3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.

## How to check in the CLI that the configuration is OK and what are the main options for?

Once the plugin is installed, log into your Centreon poller's CLI using the
**centreon-engine** user account (`su - centreon-engine`). Test that the connector
is able to monitor a resource using a command like this one (replace the sample values by yours):

```bash
/usr/lib/centreon/plugins/centreon_database_db2.pl \
	--plugin=database::db2::plugin \
	--server='10.0.0.1' \
	--port='50000' \
	--database='xxxxxx' \
	--username='' \

> 	--password=''   \

Reviewer: same as FR
